src/lwc/psElevateTokenHandler/psElevateTokenHandler.js

```diff
@@ -0,0 +1,168 @@
+
+import { fireEvent } from 'c/pubsubNoPageRef';
+import { getNamespace, isFunction, isNull } from 'c/utilCommon';
+
+import PAYMENT_AUTHORIZATION_TOKEN_FIELD from '@salesforce/schema/DataImport__c.Payment_Authorization_Token__c';
```

Comment:
The namespace is verified by checking `DataImport__c.Payment_Authorization_Token__c`. This is valid even when the user does not have read access to the NPSP Data Import SObject or read access to the field. @rponti-sforg , please verify  when testing the feature for a user that is not an admin.